RSSReader-Maruyama/DataModel/User.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  User.swift
+//  RSSReader-Maruyama
+//
+//  Created by 丸山翔 on 2020/09/03.
+//  Copyright © 2020 丸山翔. All rights reserved.
+//
+
+import UIKit
+
+class User: NSObject, Codable {
```

Comment:
```suggestion
class User: Codable {
```
NSObjectの継承は不要です。

Reply:
```suggestion
struct User: Codable {
```
データの構造を表すクラスを定義する際はstructを使用しましょう。